apps/web/src/migrations/20240403_183958.json

```diff
@@ -0,0 +1,2296 @@
+{
+  "id": "863dc6ed-c497-463d-85fc-d2c41be46c88",
```

Comment:
should we `.gitignore` the migration json files? I'm not sure i understand their use and they are always 2,000+ lines